cmd/pinning/extract.go

```diff
@@ -49,31 +50,15 @@ specify stdout. By default - is used.`)
 
 // extract will extract images from the CSV located on the path
 func extract(manifestPath string, output io.Writer) error {
-	out := []interface{}{}
-
 	log.Printf("extracting image references from %s\n", manifestPath)
 	operatorManifests, err := pullspec.FromDirectory(manifestPath, pullspec.DefaultHeuristic)
-
+	imageNames, err := image.Extract(operatorManifests)
```

Comment:
What about the err from the `pullspec.FromDirectory`. I think we're eating an error somewhere.